core/src/main/scala/scalaz/Equal.scala

```diff
@@ -86,7 +83,7 @@ object Equal {
       }
     }
   }
-  implicit val Derives: Derives[Equal] = new EqualDerives
+  implicit val equal_decidable: Decidable[Equal] = new EqualDecidable
```

Comment:
Snake_Case ? :(

Reply:
this is the convention we're be using in the `@deriving` plugin. I'd create untypeable names if I could, they are only supposed to be found implicitly.

Reply:
oh, but on Equal I could be convinced to change it to something conventional.